### DISCUSSION TITLE
Sample Link 404 Issue

### DISCUSSION THREAD

**Discussion by MillyRen** [KevinnZou/compose-webview-multiplatform#57] [General]
The link cannot enter:
For a full example, please refers to [BasicWebViewSample](https://github.com/KevinnZou/compose-webview-multiplatform/blob/main/webview/src/commonMain/kotlin/sample/BasicWebViewSample.kt)

**Comment by KevinnZou** [comment]
@MillyRen Thank you for your suggestions! I apologize for forgetting to update the link after renaming the sample module. The issue has been fixed now. I apologize for any inconvenience caused.
